docs/source/intro/datasets.rst

```diff
@@ -36,6 +36,46 @@ formats:
+
+After a labeling decision is made and the record is exported from ASReview Lab, the decision
+    (ASReview_relevant/ASReview_irrelevant/ASReview_not_seen) is stored under the N1 list-type tag:
+
+.. table:: Table with (some) RIS tag definitions
+    :widths: 20 60 20
+
+    +-------------+---------------------------------------------------------------------------------------------------------+-----------+
+    | Tag         | Meaning                                                                                                 | Mandatory |
+    +=============+=========================================================================================================+===========+
+    | TY          | Type of reference (must be the first tag)                                                               | yes       |
+    +-------------+---------------------------------------------------------------------------------------------------------+-----------+
+    | T1          | Primary title                                                                                           | yes\*     |
+    +-------------+---------------------------------------------------------------------------------------------------------+-----------+
+    | AB          | Abstract                                                                                                | yes\*     |
+    +-------------+---------------------------------------------------------------------------------------------------------+-----------+
+    | KW          | Keywords                                                                                                | no\*\*    |
+    +-------------+---------------------------------------------------------------------------------------------------------+-----------+
+    | AU          | Author                                                                                                  | no\*\*    |
+    +-------------+---------------------------------------------------------------------------------------------------------+-----------+
+    | DO          | DOI                                                                                                     | no        |
+    +-------------+---------------------------------------------------------------------------------------------------------+-----------+
+    | N1          | Notes                                                                                                   | no\*\*    |
+    +-------------+---------------------------------------------------------------------------------------------------------+-----------+
+    | ER          | End of Reference (must be empty and the last tag)                                                       | yes       |
+    +-------------+---------------------------------------------------------------------------------------------------------+-----------+
```

Comment:
Very technical, I'm not sure what the benefits are for an end-user. I would prefer to move this table to a technical section and explain to the user what to expect when importing/exporting. The user will use a ref man in 99% of the cases.

Reply:
I just followed the documentation page and what was on it previously to match the style.
Maybe we can move this table to a technical section when we overhaul the documentation for v1?

Reply:
(There is also a similar table for tabular data sets on the same page.)

Reply:
> (There is also a similar table for tabular data sets on the same page.)

Yes, but that's something that's not standardized. I don't expect any of our users to open de RIS file outside the ref man. So they will never see something like `TY` and `ER`.

Reply:
I have removed the table. It can be easily recreated if necessary at a later stage.